### PR TITLE
Homework 1 Submission

### PR DESCRIPTION
**Pull request by marieakerman**
I forked the HW1 assignment from the uc-cfss page in GitHub to my GitHub page, then cloned the assignment to my RStudio, where I saved it in a folder named HW1. I was then able to make edits, preview, and save changes. Next, I committed the changes and checked my GitHub page. There were some errors, which I then corrected in RStudio before committing the new version. Now it displayed correctly on my GitHub page. Finally, I clicked "Create a pull request" and selected "marieakerman/hw01" under "head fork," entered these comments, and clicked "Create pull request" to submit.

- I had trouble locating where I had saved my HW1 file within my computer, but Chelsea was able to help me locate it. 
- The Mastering Markdown guide was helpful with figuring out how to format lists, bold, etc., which were all relatively straightforward. 
- When I committed my changes to GitHub the image did not display, though it had when I previewed it on my computer. This was because the file name for my image had spaces in it; once I deleted the spaces the image displayed properly.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,20 @@
 # hw01
-Repository for homework 1
+
+![picture](PictureofMarie.jpg)
```
Comment (kmsurma) [inline README.md:3]: It appears that the caption you attempted to add to this photo "picture" was not added in the file on GitHub. It does, however, appear when previewed in RStudio.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,20 @@
...
+
+# Marie's Biography
```
Comment (kmsurma) [inline README.md:5]: Your headers are very well-executed and give the file a clean and concise feel. Great job!

File: README.md [hunk 3]
```diff
@@ -1,2 +1,20 @@
...
+### Hobbies and Interests
+I love to:
+
+* Bake, especially recipes from [Smitten Kitchen](https://smittenkitchen.com/)
+* Eat
+* Read
+* Sing
+* Travel
+* Bike
+* Have long conversations with friends
```
Comment (kmsurma) [inline README.md:20]: Great job on the bulleted list and link! The points are easy to read and understand, and your link works properly.

**Review by kmsurma — COMMENTED** [review]
Evaluation HW-01 for Marie Akerman:

Marie properly executed the code required to create headers, bullet points, and links. The presentation of the assignment is eye-catching and easy to read, with headers clearly labeling the material that will follow.

Something that I learned from Marie's code was how to create headers of different sizes, using a different number of pound signs. While I have no specific criticisms, I would have liked to see greater experimentation with what can be done in R Markdown, perhaps including elements in the file that were not explicitly mentioned on the homework assignment, such as video clips or sound files. 

When previewing your file, it looks like you attempted to add a caption to your picture by placing text inside the brackets in the code used for the image. 
The caption, however, did not appear on Github, while it appeared on the preview in RStudio.

**Review by amandaroseb — COMMENTED** [review]
The page is clearly organized and formatted with two different sized headers, bullets, some bolded words, an image and a working  link to your blog. Nice job. 

It sounds like you went through some troubleshooting exercises with differences between Github and RStudio, but you ultimately solved the problem of making your image appear. You might consider what a meaningful caption might add instead of "picture", so that when users scroll over the picture they see something like the name of the location where the picture was taken, or the person pictured (e.g. your name) or something like that. In case you aren't sure, to get mine to show the caption on Github I used the syntax with the caption in parentheses (noted in the code).

Looking at your work prompted me to go back and learn more about the differences between markdown syntax and Github flavored markdown - which was useful! Nice job noticing differences between the Rstudio preview and the Github preview, and working to fix them.

File: README.md (hunk 1 above)
Comment (nnickels) [inline README.md:3]: Great job. The [picture] in brackets actually does not appear by default; there are other options for adding a caption or scroll over descriptive text.

**Review by nnickels — COMMENTED** [review]
Evaluation HW-01 for Marie Akerman
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |     X    |              |            |
| **Coding strategy**         |     X      |              |            |
| **Presentation**            |     X     |              |            |
| **Achievement, creativity** |           |      X      |            |
| **Ease of access**          |     X      |              |            |

Remarks:

* Great job on the first homework! You've displayed mastery in several markdown elements.
* Nice job saving an image to your repo and calling its file path from there.

**Pull request closed without merging**